lib/elixir/lib/calendar.ex

```diff
@@ -576,6 +600,34 @@ defmodule Time do
     end
   end
 
+  @doc """
+  Compares two `Time` structs. Returns :gt if first time is later than the second
+  and :lt for vice versa. If the two times are equal :eq is returned
+
+  ## Examples
+
+    iex> Time.compare(Time.from_erl!({16, 4, 16}), Time.from_erl!({16, 4, 28}))
+    :lt
+  """
+  @spec compare(Time.t, Time.t) :: :gt | :eq | :lt
```

Comment:
could we sort the specs from low to high? i.e. `:lt | :eq | :gt` just for the sake of keeping a logical order